README.md

```diff
@@ -325,16 +327,18 @@ const s = new Script(`
   ++this.ran;
 `);
 
-dom.runVMScript(s);
-dom.runVMScript(s);
-dom.runVMScript(s);
+const vmContext = dom.getInternalVMContext();
+
+script.runInContext(vmContext);
+script.runInContext(vmContext);
+script.runInContext(vmContext);
```

Comment:
Actually, the&nbsp;variable is&nbsp;named&nbsp;`s` instead&nbsp;of&nbsp;`script`.

Reply:
Sigh, thanks. I'll do a force-push.